client/my-sites/theme/main.jsx

```diff
@@ -1555,12 +1545,12 @@ const ThemeSheetWithOptions = ( props ) => {
 		! ( isSiteWooExpressFreeTrial && isThemeBundleWooCommerce )
 	) {
 		defaultOption = 'upgradePlanForBundledThemes';
+	} else if ( themeTier?.slug === 'community' && ! canInstallThemes ) {
```

Comment:
I assume FEATURE_INSTALL_THEMES is the correct thing to check here, but there is a lot of context in these theme files so please let me know if something makes more sense. 

Its my understanding we need install permissions to activate a community theme, and the old CTA for community themes upgrade redirected to the install page after checkout.

Reply:
I guess my confusion is the difference between `canInstallThemes` and `canUploadThemes`. The flow made it seem like install made more sense, but I saw the old modal permission checking the upload. Im fuzzy on the difference here, are they technically the same and just a difference between whether they get them from the showcase or upload a file manually?

Reply:
> just a difference between whether they get them from the showcase or upload a file manually?

Yes, there are different `WPCOM_Features` for installing themes and uploading a theme as a zip file.

(Our own first party themes are installed on the multisite already, individual sites just activate them, but they too have their own WPCom_Feature)